src/senselab/utils/tasks/chunking.py

```diff
@@ -0,0 +1,25 @@
+"""This module provides functionality for chunking text into overlapping segments."""
+
+from typing import List
+
+from transformers import AutoTokenizer
+
+
+def chunk_text(text: str, tokenizer: AutoTokenizer, max_length: int, overlap: int) -> List[str]:
```

Comment:
1. this is a text-specific utility and not a general utility.
2. I think chunking may be already managed inside of the text pipeline (at least for HF)